### PR TITLE
SSL consistency renderer

### PR DESCRIPTION
**Pull request by pierky**

### CODE REVIEW

File: ripe/atlas/tools/commands/report.py [hunk 1]
```diff
@@ -49,22 +49,23 @@ def get_probes(self):
...
-        detail = AtlasRequest(url_path=self.URLS["detail"].format(pk)).get()[1]
         latest = AtlasRequest(url_path=latest_url).get()[1]
 
         if not latest:
             raise RipeAtlasToolsException(
                 "There aren't any results available for that measurement")
 
-        renderer = Renderer.get_renderer(
-            self.arguments.renderer, detail["type"]["name"])()
-
```
Comment (astrikos) [inline ripe/atlas/tools/commands/report.py]: Does this change help somewhere?

Comment (pierky) [inline ripe/atlas/tools/commands/report.py]: Abort ASAP if the renderer can't handle the type of the requested measurement.

Comment (astrikos) [inline ripe/atlas/tools/commands/report.py]: ok makes sense :)

**Comment by danielquinn** [comment]
As this list is growing, I may have to rethink the directory structure some.  Perhaps something like this:

    --renderer=ssl/consistency
    [Loads renderers.ssl.consistency]

I'm not sure yet, but I just thought you might want to be kept in the loop.

That part aside, this is great work, thank you!

**Pull request merged**